### PR TITLE
{math}[foss/2017b,intel/2017b] gmpy2 v2.0.8 w/ Python 2.7.14 + 3.6.3

### PR DESCRIPTION
**Pull request by murfreesboro**
(created using `eb --new-pr`)
require the PR ~#8412~

### CODE REVIEW

**Comment by murfreesboro** [comment]
Test report by @murfreesboro
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
dev806.vampire - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.7.5
See https://gist.github.com/35037932a9f97fda4dfdb703ef30a3de for a full test report.

**Comment by vanzod** [comment]
Re-trigger Travis

File: easybuild/easyconfigs/g/gmpy2/gmpy2-2.0.8-foss-2017b-Python-2.7.14.eb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+sources = [SOURCE_ZIP]
+checksums = ['dd233e3288b90f21b0bb384bcc7a7e73557bb112ccf0032ad52aa614eb373d3f']
+
+dependencies = [
+    ('Python', '2.7.14'),
+    ('GMP', '6.1.2'),
+    ('MPFR', '3.1.6'),
+    ('MPC', '1.0.3', '-MPFR-3.1.6'),
+]
+
```
Comment (vanzod) [inline easybuild/easyconfigs/g/gmpy2/gmpy2-2.0.8-foss-2017b-Python-2.7.14.eb]: Please add:
```
use_pip = True
download_dep_fail = True
```

File: easybuild/easyconfigs/g/gmpy2/gmpy2-2.0.8-foss-2017b-Python-3.6.3.eb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+sources = [SOURCE_ZIP]
+checksums = ['dd233e3288b90f21b0bb384bcc7a7e73557bb112ccf0032ad52aa614eb373d3f']
+
+dependencies = [
+    ('Python', '3.6.3'),
+    ('GMP', '6.1.2'),
+    ('MPFR', '3.1.6'),
+    ('MPC', '1.0.3', '-MPFR-3.1.6'),
+]
+
```
Comment (vanzod) [inline easybuild/easyconfigs/g/gmpy2/gmpy2-2.0.8-foss-2017b-Python-3.6.3.eb]: Please add:
```
use_pip = True
download_dep_fail = True
```

File: easybuild/easyconfigs/g/gmpy2/gmpy2-2.0.8-intel-2017b-Python-2.7.14.eb [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+sources = [SOURCE_ZIP]
+checksums = ['dd233e3288b90f21b0bb384bcc7a7e73557bb112ccf0032ad52aa614eb373d3f']
+
+dependencies = [
+    ('Python', '2.7.14'),
+    ('GMP', '6.1.2'),
+    ('MPFR', '3.1.6'),
+    ('MPC', '1.0.3', '-MPFR-3.1.6'),
+]
+
```
Comment (vanzod) [inline easybuild/easyconfigs/g/gmpy2/gmpy2-2.0.8-intel-2017b-Python-2.7.14.eb]: Please add:
```
use_pip = True
download_dep_fail = True
```

File: easybuild/easyconfigs/g/gmpy2/gmpy2-2.0.8-intel-2017b-Python-3.6.3.eb [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+sources = [SOURCE_ZIP]
+checksums = ['dd233e3288b90f21b0bb384bcc7a7e73557bb112ccf0032ad52aa614eb373d3f']
+
+dependencies = [
+    ('Python', '3.6.3'),
+    ('GMP', '6.1.2'),
+    ('MPFR', '3.1.6'),
+    ('MPC', '1.0.3', '-MPFR-3.1.6'),
+]
+
```
Comment (vanzod) [inline easybuild/easyconfigs/g/gmpy2/gmpy2-2.0.8-intel-2017b-Python-3.6.3.eb]: Please add:
```
use_pip = True
download_dep_fail = True
```

File: easybuild/easyconfigs/g/gmpy2/gmpy2-2.0.8-foss-2017b-Python-2.7.14.eb (hunk 1 above)
Comment (murfreesboro) [reply easybuild/easyconfigs/g/gmpy2/gmpy2-2.0.8-foss-2017b-Python-2.7.14.eb]: added the options.

File: easybuild/easyconfigs/g/gmpy2/gmpy2-2.0.8-intel-2017b-Python-3.6.3.eb (hunk 4 above)
Comment (murfreesboro) [reply easybuild/easyconfigs/g/gmpy2/gmpy2-2.0.8-intel-2017b-Python-3.6.3.eb]: add the options.

File: easybuild/easyconfigs/g/gmpy2/gmpy2-2.0.8-intel-2017b-Python-2.7.14.eb (hunk 3 above)
Comment (murfreesboro) [reply easybuild/easyconfigs/g/gmpy2/gmpy2-2.0.8-intel-2017b-Python-2.7.14.eb]: add the options.

File: easybuild/easyconfigs/g/gmpy2/gmpy2-2.0.8-foss-2017b-Python-3.6.3.eb (hunk 2 above)
Comment (murfreesboro) [reply easybuild/easyconfigs/g/gmpy2/gmpy2-2.0.8-foss-2017b-Python-3.6.3.eb]: add the options.

**Comment by murfreesboro** [comment]
Test report by @murfreesboro
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
dev806.vampire - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.7.5
See https://gist.github.com/4c845b279cb10330902027e1ba66e3d0 for a full test report.

**Comment by vanzod** [comment]
Re-trigger Travis

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 5 out of 5 (4 easyconfigs in this PR)
cermis - Linux debian 9.9, Intel(R) Xeon(R) CPU           X5670  @ 2.93GHz, Python 2.7.13
See https://gist.github.com/ebdf530c82ed87e46891040436fcf7d5 for a full test report.

**Comment by vanzod** [comment]
Going in, thanks @murfreesboro!

**Pull request merged**